great_expectations/expectations/expectation.py

```diff
@@ -1632,20 +1634,46 @@ def _validate_metric_value_between(  # noqa: C901, PLR0912
         return {"success": success, "result": {"observed_value": metric_value}}
 
 
+UNEXPECTED_ROWS_EXPECTATION_SHORT_DESCRIPTION = (
+    "This Expectation will fail validation if the query returns one or more rows.  "
+    "The WHERE clause defines the fail criteria."
+)
+UNEXPECTED_ROWS_QUERY_DESCRIPTION = "A SQL or Spark-SQL query to be executed for validation."
+SUPPORTED_DATA_SOURCES = ["SQL", "Spark-SQL"]
```

Comment:
This should be left as an empty list `[]`. This is probably supported by many (if not all) OSS data sources, but I'm not sure what level of testing has been done, and this metadata field isn't used anywhere today. The level of abstraction that we use for this field is more like `["Snowflake", "Postgres", ...]`.

Reply:
You can just get rid of the variable and pass `[]` in the `schema_extra` like you did for `data_quality_issues`.